Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-wasmtime = "11.0.1"
+wasmtime = "22.0.0"
```

Comment:
This should be a separate PR, since it's unrelated to the buffer types change.

This is already done in #20, but needs updating...

Reply:
OK, thanks @PiotrSikora . I will wait for #20 to be merged, then pull the changes into this PR.

Reply:
It's merged now. Could you update this PR?

Reply:
Done. Thanks